public/js/applicantModal.js

```diff
@@ -49,11 +55,38 @@ function copyToClipboard(element) {
     document.querySelector("button.clipboard").innerText = 'Copied';
 }
 
-export function addEventListenersToDisplayApplicantModal() {
+function showAlertforApptitudeButton(alertSelector)
+{
+    document.querySelector(alertSelector).classList.remove('hidden')
+    setTimeout(() => {
+        document.querySelector(alertSelector).classList.add('hidden')
+    }, 3000)
+}
+
+function aptitudeScoreButtonClick(e)
+{
+    fetch(`/api/getAptitudeScore?email=${e.target.dataset.email}`)
+        .then(response => response.json())
+        .then(data => {
+            if (data.success) {
+                document.querySelector('#aptitude').textContent = `${data.data.score}%`;
+                showAlertforApptitudeButton('#aptitudeScoreSuccess')
+            } else {
+                document.querySelector('#aptitudeScoreError').textContent = data.message;
+                showAlertforApptitudeButton('#aptitudeScoreError')
+            }
+            })
+            .catch(error => {
+                console.error('Error:', error);
```

Comment:
this is not an appropriate thing to do with an error. you should handle this error the same way as the data.success failure, show it to the user as "unexpected error"